codec/depinject.go

```diff
@@ -1,12 +1,18 @@
 package codec
 
 import (
+	"fmt"
+
 	"github.com/cosmos/gogoproto/proto"
 
+	authmodulev1 "cosmossdk.io/api/cosmos/auth/module/v1"
+	stakingmodulev1 "cosmossdk.io/api/cosmos/staking/module/v1"
```

Comment:
are we fine with this for now? I guess there isn't a choice in the current design? maybe we open an issue to close before we release?